cupyx/scipy/signal/_spectral_impl.py

```diff
@@ -0,0 +1,205 @@
+"""
+Spectral analysis functions and utilities.
+
+Some of the functions defined here were ported directly from CuSignal under
+terms of the Apache license, under the following notice
+
+Copyright (c) 2019-2020, NVIDIA CORPORATION.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
```

Comment:
Would you update this part to reflect cuSignal's updated MIT license?
https://github.com/rapidsai/cusignal/pull/587/files#diff-c693279643b8cd5d248172d9c22cb7cf4ed163a3c98c8a3f69c2717edd3eacb7